app/assets/javascripts/components/campaign/campaign.jsx

```diff
@@ -47,7 +47,9 @@ export const Campaign = createReactClass({
         <div className="container campaign_main">
           <section className="overview container">
             <CampaignStats campaign={this.props.campaign} />
-            {this.props.campaign.course_stats && <WikidataOverviewStats
+            {this.props.campaign.course_stats &&
```

Comment:
This property should be included in the reducer `app/assets/javascripts/reducers/campaign.js`